packages/@aws-cdk/aws-cloudfront/lib/origin.ts

```diff
@@ -200,10 +210,9 @@ export abstract class OriginBase implements IOrigin {
   /**
    * Takes origin shield region and converts to CfnDistribution.OriginShieldProperty
    */
-  private renderOriginShield(originShieldRegion?: string): CfnDistribution.OriginShieldProperty | undefined {
-    return originShieldRegion
-      ? { enabled: true, originShieldRegion }
-      : undefined;
+  private renderOriginShield(originShieldRegion?: string, originShieldEnabled?: boolean ): CfnDistribution.OriginShieldProperty | undefined {
+    if (originShieldEnabled === false) return { enabled: false };
+    return originShieldRegion ? { enabled: true, originShieldRegion } : undefined;
```

Comment:
Totally okay in principal. We avoid single line `if` statements. In combination with my other suggestion it could look like this:

```suggestion
  private renderOriginShield(originShieldEnabled: boolean, originShieldRegion?: string): CfnDistribution.OriginShieldProperty | undefined {
    if (!originShieldEnabled) {
      return { enabled: false };
    }
    return originShieldRegion ? { enabled: true, originShieldRegion } : undefined;
```